### PR TITLE
Added guide for plotly

### PR DESCRIPTION
**Pull request by sourabh1031**
NOTE:
- Please fill out the template below 

## Did you read the instructions in the [README](https://github.com/the-ethan-hunt/first-timers-guide/blob/master/README.md)?

> Y

## Have you read the instructions in [CONTRIBUTING](https://github.com/the-ethan-hunt/first-timers-guide/blob/master/CONTRIBUTING.md)?

> Y

## Have you written this article by yourself? Have you taken inspiration from anywhere and then written it in your words?

> [I have taken the reference from [wikipedia](https://en.wikipedia.org/wiki/Plotly), and [analytics vidhya](https://www.analyticsvidhya.com/blog/2017/01/beginners-guide-to-create-beautiful-interactive-data-visualizations-using-plotly-in-r-and-python/) ]

### CODE REVIEW

File: guides/Visual_design/Data visualisation/plotly.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+plotly.js and plotly.py are high-level, declarative charting libraries. They ship with over 30 chart types, including scientific charts, 3D graphs, statistical charts, SVG maps, financial charts, and more.
+### falcon-sql-client
+Falcon is a free, open-source SQL editor with inline data visualization.
+### react-plotly.js
+A plotly.js React component from Plotly. The basis of Plotly's React component suite.
+### react-chart-editor
+Customizable React-based editor panel for Plotly charts, Part of Plotly's React Component Suite for building data visualization Web apps and products.
+
+# Get hands on with learning
+Here are few referral links which provide a good documentation for learning
```
Comment (RN0311) [inline guides/Visual_design/Data visualisation/plotly.md]: Can you please add break statement, while providing links.On same lines, it looks cluttered.
Please use this [cheatsheet](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet), to know about how to add bullets.

File: guides/Visual_design/Data visualisation/plotly.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Plotly
+
+# Introduction
+Plotly from the name it gives clear indication of it what it provides.The logical breakdown of the name tells something which is related to plotting something(i.e graphs charts or visual graphics) isn't ?.A formal description of plotly is described as follows
+
+# Description
+Plotly is one of the data visualization and data analytics tools.It is built on top of data visualization library D3.js, HTML and CSS. It is developed using Python and  Django. It is compatible with a number of languages/ tools: R, Python, MATLAB, Perl, Julia, Arduino. Plotly provides online graphing and statistics tools for individuals and collaborations .It also offers scientific graphing libraries for Python, R, MATLAB, Perl, Julia, and REST.The github repo of plotly is found [here](https://github.com/plotly).
+
+It provides six main products
```
Comment (RN0311) [inline guides/Visual_design/Data visualisation/plotly.md]: Not "products" but features, I guess.

File: guides/Visual_design/Data visualisation/plotly.md (hunk 1 above)
Comment (sourabh1031) [reply guides/Visual_design/Data visualisation/plotly.md]: @RN0311 if I add break statement after the word then it  would not  redirect to the specific link .Do you want me to add as such?

Comment (the-ethan-hunt) [reply guides/Visual_design/Data visualisation/plotly.md]: The links will not be broken. But bulleting is a must here.

**Comment by sourabh1031** [comment]
@the-ethan-hunt  i have updated please review

**Pull request merged**